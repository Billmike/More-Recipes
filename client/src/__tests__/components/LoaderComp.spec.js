import React from 'react';
import { shallow } from 'enzyme';
import LoaderComp from '../../components/LoaderComp';

describe('<Modal />', () => {
  it('Should render the Modal correctly', () => {
    const wrapper = shallow(<LoaderComp />);
    expect(wrapper).toMatchSnapshot();
  });
});
