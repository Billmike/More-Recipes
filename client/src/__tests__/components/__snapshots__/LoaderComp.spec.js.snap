// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> Should render the Modal correctly 1`] = `
<div>
  <Component
    className="spinner"
    color="#5f9ea0"
    component="div"
    corners={1}
    direction={1}
    hwaccel={false}
    left="50%"
    length={20}
    lines={13}
    loadedClassName="loadedContent"
    parentClassName="loader"
    radius={30}
    rotate={0}
    scale={1}
    shadow={false}
    speed={1}
    top="50%"
    trail={60}
    width={10}
    zIndex={2000000000}
  />
</div>
`;
